ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -5467,7 +5523,19 @@ CHWVM:
   implements:
   - OPERATIONAL
 
-
+CHWVVM:
+  guid: "cd1be55d-85bc-4e0b-82b0-f3e38c427521"
+  description: "Chilled water valve voltage monitoring."
+  is_abstract: true
+  opt_uses:
+  - chilled_water_flowrate_sensor
+  - chilled_water_valve_voltage_sensor
```

Comment:
Make the sensor required. Please confirm if this valve controls to any air temperature.

Reply:
Made the sensor required. 
Also no way to confirm if valve controls air temperature as we don't have access to BAS.

Reply:
Sorry, to clarify please make the voltage sensor required rather than the flowrate sensor.

Reply:
@shambergoldstein Done